build.sbt

```diff
@@ -279,7 +279,7 @@ lazy val scioCore: Project = Project(
     "com.google.guava" % "guava" % guavaVersion,
     "com.google.protobuf" % "protobuf-java" % protobufVersion,
     "me.lyh" %% "protobuf-generic" % protobufGenericVersion,
-    "org.apache.xbean" % "xbean-asm5-shaded" % asmVersion
+    "org.apache.xbean" % "xbean-asm6-shaded" % asmVersion
```

Comment:
I'd wait till @andrewsmartin can sanity check closure cleaner.

Reply:
@nevillelyh yeah this requires more changes and i first thought.

Reply:
@nevillelyh postponing this update